Source/CombatExtended/Compatibility/PawnKindAutoPatcher.cs

```diff
@@ -16,7 +16,7 @@ public class PawnKindPatcher
         static PawnKindPatcher()
         {
            
-            List<PawnKindDef> stuff = DefDatabase<PawnKindDef>.AllDefsListForReading.FindAll(i => i.modExtensions?.Any(tt => !(tt is LoadoutPropertiesExtension)) ?? true && i.RaceProps.Animal == false);
+            List<PawnKindDef> stuff = DefDatabase<PawnKindDef>.AllDefsListForReading.FindAll(i => i.modExtensions?.Any(tt => !(tt is LoadoutPropertiesExtension)) ?? true && i.RaceProps.Animal == false && (i.race?.comps?.Any(t => t is CompProperties_Inventory) ?? false));

```

Comment:
I feel like this expression is getting a bit too unruly, if you care enough, could you split the conditions into their own variables and then `&&` those?

Reply:
sure I guess